src/services/ui/src/features/package-actions/lib/modules/respond-to-rai/waiver/b-waiver-rai.tsx

```diff
@@ -30,11 +30,10 @@ export const bWaiverRaiFields: ReactElement[] = [
     key={"field-attachments"}
     attachments={[
       {
-        name: "raiResponseLetter",
+        name: "raiResponseLetter1915b",
```

Comment:
raiResponseLetter1915b -> raiResponseLetterWaiver makes more sense to me :)

Reply:
okay I changed it :)